doc/help.md

```diff
@@ -4,7 +4,7 @@ The **help** command of the [Winget-Create](../README.md) tool displays help for
 
 ## Usage
 
-* Display help for all commands: `WingetCreateCLI.exe  -?`
-* View options for a command: `WingetCreateCLI.exe  <command> -?`
+* Display help for all commands: `wingetcreate  -?`
+* View options for a command: `wingetcreate  <command> -?`
```

Comment:
We would still prefer to keep the `.exe` extension in the usage examples. Could you change this to `wingetcreate.exe`